autoload/lsp/lsp.vim

```diff
@@ -832,6 +832,16 @@ export def ShowReferences(peek: bool)
   lspserver.showReferences(peek)
 enddef
 
+# send custom locations request
+def g:FindLocations(peek: bool, method: string, args: dict<any> = {})
+  var lspserver: dict<any> = buf.CurbufGetServerChecked('references')
```

Comment:
this 'references' input maybe not accurate, which seems your 'method' were varied.